src/test/java/com/puppycrawl/tools/checkstyle/api/ViolationTest.java

```diff
@@ -20,17 +20,25 @@
 package com.puppycrawl.tools.checkstyle.api;
 
 import static com.google.common.truth.Truth.assertWithMessage;
+import static com.puppycrawl.tools.checkstyle.checks.modifier.InterfaceMemberImpliedModifierCheck.MSG_KEY;
 import static com.puppycrawl.tools.checkstyle.utils.CommonUtil.EMPTY_OBJECT_ARRAY;
 
 import org.junit.jupiter.api.Test;
 import org.junitpioneer.jupiter.DefaultLocale;
 
+import com.puppycrawl.tools.checkstyle.AbstractModuleTestSupport;
 import com.puppycrawl.tools.checkstyle.checks.annotation.AnnotationLocationCheck;
 import com.puppycrawl.tools.checkstyle.checks.annotation.AnnotationOnSameLineCheck;
+import com.puppycrawl.tools.checkstyle.checks.modifier.InterfaceMemberImpliedModifierCheck;
 import nl.jqno.equalsverifier.EqualsVerifier;
 import nl.jqno.equalsverifier.EqualsVerifierReport;
 
-public class ViolationTest {
+public class ViolationTest extends AbstractModuleTestSupport {
```

Comment:
Violation class is not a module.

can you move test to InterfaceMemberImpliedModifierCheckTest?
and just execution InterfaceMemberImpliedModifierCheckTest in api group of pitest.

Reply:
This test case was just found by just hit and trial by run all the check in local for API group and generating pit report.

I have include this test so that for a single test we can avoid all test execution.

Reply:
I understand this.
I hope one day we will find a way to run single test by pitest config.
This is good feeback point to pitest project.

Reply:
I added it to https://github.com/checkstyle/checkstyle/issues/13142